New Entry: R loops (#878)
Codecademy/docs: content/r/concepts/loops/loops.md

---
Title: 'Loops'

> Description: 'A loop is used to perform some operation as long as a declared condition is truthy.'

Reviewer: ```suggestion
Description: 'A loop is used to repeat some operation a multiple number of times.'
```

Reviewer: The original description applied to `while` loops, but not `for` loops.

Subjects:
  - 'Computer Science'
  - 'Data Science'
Tags:
  - 'Control Flow'
  - 'Iterators'
  - 'Loops'
CatalogContent:
  - 'learn-r'
  - 'paths/computer-science'
---

> A loop is used to perform some operation as long as a declared condition is truthy. In R, there are `while` and `for` loops.

Reviewer: ```suggestion
A loop is used to repeat some operation a multiple number of times. In R, there are `while` and `for` loops.
```

Reviewer: Again, it needs to apply to `for` as well as `while` loops.

> A loop is used to perform some operation given that a declared condition is truthy. In R, there are `while` and `for` loops.

Reviewer 2: ```suggestion
A loop is used to perform some operation as long as declared condition is truthy. In R, there are `while` and `for` loops.
```

## While Loop

> The statement(s) within a `while` loop code block will continue to execute given that a declared condition is `TRUE`.

Reviewer: ```suggestion
The statement(s) within a `while` loop code block will continue to execute as long as a declared condition is `TRUE`.
```
